CodingStyle

```diff
@@ -28,12 +28,12 @@ by section.
 
     Google uses CamelCaps for all type names.  We use two naming schemes:
 
-    - for naked structs (simple data containers), lower case with _d
-      suffix ('d' for data).  Not _t, because that means typdef.
+    - for naked structs (simple data containers), lower case with _t.
```

Comment:
It seems that the spirit of this would extend to tuples and discriminated unions? If so, it might be worth making a note to that effect.

Reply:
For example, should we write:
using foo_t = tuple<int, float, string>;
using bar_t = variant<int, float, string>;
The resultant types are certainly not "naked", in the sense that they have lots of member functions, etc., but they are also certainly "simple data containers". I'm cool either way: I'm not sure we need to have a rule to _t or not to _t, but that /is/ the question! (I said there would be puns... didn't say they'd be good!)